mediafile.py

```diff
@@ -1643,6 +1643,12 @@ def update(self, dict):
         StorageStyle('ARTIST'),
         ASFStorageStyle('Author'),
     )
+    artists = ListMediaField(
+        #MP3ListDescStorageStyle(desc='ARTISTS'),
```

Comment:
How can I do something like this?
I've tag like:
```
'TXXX:ARTISTS': TXXX(encoding=<Encoding.UTF16: 1>, desc='ARTISTS', text=['Miuosh/Jimek/Narodowa Orkiestra Symfoniczna Polskiego Radia w Katowicach']),
```
And it's supposed to be interpreted as list: ['Miuosh', 'Jimek']

Reply:
Source for this tag: https://picard-docs.musicbrainz.org/en/appendices/tag_mapping.html#artists

Reply:
@sampsyo that's why it's disabled (answer to review below).
I don't know how to handle this.
There's no class like that and I don't know if I need to write mine (would like not to) or if there's any implementation.

Reply:
I've more problems:
I mentioned `TXXX:ARTISTS` consisting of array of artists but...
I also have `TXXX:ARTISTS` consisting of one artist but also containing unescaped `\`:
` 'TXXX:ARTISTS': TXXX(encoding=<Encoding.UTF16: 1>, desc='ARTISTS', text=['AC/DC']),`

Reply:
This seems to be similar issue to problem I've encountered in #43 
I need some advice about this.